core-ui/src/shared/components/ResourcesList/ResourcesList.js

```diff
@@ -237,6 +237,7 @@ export function ResourceListRenderer({
       nameSelector(resource) + '.yaml',
       handleSaveClick(resource),
       isProtected(resource),
+      isProtected(resource),
```

Comment:
why twice?

Reply:
Because we are setting `readOnly` and `isProtected` parameters, it's the same in this case.